stateproof/worker_test.go

```diff
@@ -62,9 +62,15 @@ type testWorkerStubs struct {
 	txmsg                 chan transactions.SignedTxn
 	totalWeight           int
 	deletedStateProofKeys map[account.ParticipationID]basics.Round
+	version               protocol.ConsensusVersion
 }
 
 func newWorkerStubs(t testing.TB, keys []account.Participation, totalWeight int) *testWorkerStubs {
+	version := config.Consensus[protocol.ConsensusCurrentVersion]
+	return newWorkerStubsWithVersion(t, keys, protocol.ConsensusCurrentVersion, version, totalWeight)
```

Comment:
I don't think you need to pass both the consensus parameters and the consensus version name.